text/0199-support-elastic-common-schema-in-opentelemetry.md

```diff
@@ -0,0 +1,265 @@
+# Support Elastic Common Schema in OpenTelemetry
+
+Collaborators: Alolita Sharma (OTel GC, Apple), Cyrille Le Clerc (Elastic), Christian Beedgen (Sumo), Jonah Kowall (Logz.io), Tigran N. (Splunk)
+
+## Introduction
+
+This proposal is to add support for the Elastic Common Schema (ECS) in the OpenTelemetry specification and provide full interoperability for ECS in OpenTelemetry component implementations. We propose to implement this support by enriching OpenTelemetry Semantic Conventions with ECS fields. The goal is to merge ECS into OTel Semantic Conventions.
```

Comment:
> The goal is to merge ECS into OTel Semantic Conventions.

What do you mean by "merge"? Is the ECS project going to be retired once the "merge" is accomplished?

Reply:
@yurishkuro There has been some confusion around this - we'll be updating the Otep to clarify the intent. Our goal is partner with the Otel community, to facilitate the adoption of ECS fields within Otel Semantic Conventions. Otel can decide which fields are appropriate to adopt or exclude. ECS will continue to be maintained by Elastic and heavily used within Kibana. However, we'd be happy for Otel members to engage in our [RFC process](https://github.com/elastic/ecs/blob/main/rfcs/PROCESS.md) to ensure ECS and Semantic Conventions are as aligned as possible going forward.